scripts/layout-initrd

```diff
@@ -29,9 +29,7 @@ ln -s usr/bin/ros              ${INITRD_DIR}/init
 ln -s bin                      ${INITRD_DIR}/usr/sbin
 ln -s usr/sbin                 ${INITRD_DIR}/sbin
 
-curl -SL ${!SYSTEM_DOCKER_URL} | tar --strip-components=1 -xzvf - -C ${INITRD_DIR}/usr/bin/
-# we have diabled the user-proxy so we get rid of system-docker-proxy
-rm -f ${INITRD_DIR}/usr/bin/system-docker-proxy
+curl -SL ${!DOCKER_URL} | tar --strip-components=1 -xzvf - -C ${INITRD_DIR}/usr/bin/
```

Comment:
I think that we should actually extract these to `${INITRD_DIR}/usr/share/ros/docker` and then add symlinks like these (I'm not sure if we need all of them but it is easier create them all first):
```bash
ln -s usr/bin/system-dockerd ${INITRD_DIR}/usr/share/ros/docker/dockerd
ls -s containerd ${INITRD_DIR}/usr/share/ros/docker/containerd
ls -s containerd-shim ${INITRD_DIR}/usr/share/ros/docker/containerd-shim
ls -s containerd-shim-runc-v2 ${INITRD_DIR}/usr/share/ros/docker/containerd-shim-runc-v2
ls -s ctr ${INITRD_DIR}/usr/share/ros/docker/ctr
ls -s docker-init ${INITRD_DIR}/usr/share/ros/docker/docker-init
ls -s docker-proxy ${INITRD_DIR}/usr/share/ros/docker/docker-proxy
ls -s runc ${INITRD_DIR}/usr/share/ros/docker/runc
```
and sh script with name `system-docker` which run `/usr/share/ros/docker/docker` with parameter `-H unix:///var/run/system-docker.sock`.

That way you don't need replace all those system-docker references and it will allow us to remove second copy of docker binaries from ISO file because we can just run system-docker build against of those same files.

Reply:
Will create the symlinks and the script as soon as I get to a console.

Reply:
There is less system-docker references in code then those symlinks

Reply:
Yea but I try avoid changes those core parts of OS which I'm not fully aware of.

And I really would like to try to create version where Docker binaries exist only once on ISO file. It will be much easier if we have those already on dedicated folder.